### ISSUE TITLE
Add support for S-series pumps and myUplink

### ISSUE THREAD

**Issue by MasterV** [emilohman/homey-nibe#33]
Nibe S-series pumps use the newer ‘myUplink’ service and can’t be added through this app. (Or any other)

Is there a way for the developer to add support for the service and s-series pumps?

**Comment by maikie1978** [comment]
Yes same issue here, I also want to integrate but its only working with Nibe Uplink and not with MyUplink. 
If somenone can add this I can continue with integrating this device in my Homey

**Comment by ackerk** [comment]
Same here. And looks like Nibe will be switching from Nibe Uplink to MyUplink for all their heatpumps in the near future, so support  for myUplink will pretty much be mandatory in order for this app to be able to continue working properly.

**Comment by cillianomongain** [comment]
Same here, would love support for MyUplink.

**Comment by Jeronimus76** [comment]
Same here. Tried everything else.
My entire new home is connected to Homey Pro, except for the Nibe S-line.
☹️

**Comment by lovegren22** [comment]
How can we support someone to assist in this? If its about time maybe a collection of money is in its order to motivate?

**Comment by MasterV** [comment]
> How can we support someone to assist in this? If its about time maybe a collection of money is in its order to motivate?

I’m up for that. Not sure how to proceed, but I’d happily donate/pay.

**Comment by Aborgh** [comment]
It seems like @emilohman hasn't replied to several of these threads. I was first thinking of contributing to this repo but I think I'll try making a new app for myUplink instead

**Comment by emilohman** [comment]
@Aborgh You are more than welcome to contribute with myUplink support!
I'm not answering because I build this app on my spare time and I don't have an S-serie pump myself and it's quite expensive to just by one...

But last week I got an email from Nibe that my heatpump is ready to be migrated to the myUplink api.. so soon when I have time I maybe look into it..

**Comment by Aborgh** [comment]
Cool thanks! I understand completely 😊. I wasn't inferring anything from my comment. 

Since homey hasn't got any support for 2 oauth endpoints it will be quite tricky to support myUplink and nibeuplink. Don't you think? 🤔 
Since all nibeuplink accounts is to migrate to myUplink , maybe a new homey app (different app id) on a new branch called "myUplink" with the new endpoint would be sufficient? 

Then merge it to main when all accounts has been migrated?

What do you think @emilohman?
